### ISSUE TITLE
Feature Request: join pages while maintaining layers

### ISSUE THREAD

**Issue by ReubenM** [qpdf/qpdf#9]
Layers currently get lost when joining pages from different files. Would be nice to be able to join pages, with different layers in each, into a new pdf that contains a super-set of all the layers from the input pages.

**Comment by jberkenbilt** [comment]
Note to self: file saved as `~/source/qpdf/misc/issue-9-Layers_1-2.pdf`

**Comment by Trevor-** [comment]
Was the issue of preserving layers "fixed" or did anyone find an alternative CLI for mergine PDFs and preserving the  layers?

**Comment by jberkenbilt** [comment]
I haven't prioritized working on this issue so far. @Trevor- can you tell me how you see layers? What viewing software do you use, and how do you turn them on and off? I haven't assessed how difficult this would be to implement, but I'm guessing it won't be that hard. I've been making time every few months for qpdf enhancements. Maybe I can try to work this in. No promises.

**Comment by mtoftdal** [comment]
@jberkenbilt , there's a Navigation Pane in Acrobat Reader called Layers. Here you can see a list of all the layers and toggle their visibility.

**Comment by Trevor-** [comment]
Yes that's right, I'm sorry I didn't get back, I mean't to and it slipped my mind

**Comment by jberkenbilt** [comment]
Okay, thanks. There's going to be some work on forms this summer, and I may be adding some higher level interfaces to the qpdf library. Part of that work may include improvements in areas like this, but I'm not sure. Anyway, I understand what this issue is now.

**Comment by jberkenbilt** [comment]
I don't want to get hopes up, but I'm starting to chip away at preserving document-level features when splitting and merging files. This one is not going to make the cut for 10.2, but I hope it will be in the not-too-distant future. Anyway, I am actually working on this and other related issues now.

**Comment by jberkenbilt** [comment]
This issue is part of the qpdf pages epic. If you are interested in following, please see #1104.
